app/services/transform_notification.rb

```diff
@@ -2,25 +2,19 @@
 
 # Sends SNS messge that says a transform was requested
 class TransformNotification
-  def self.publish
-    new.publish
+  def self.publish(data_dir)
+    new.publish(data_dir)
   end
 
-  def publish
+  def publish(data_dir)
     client.publish(
       topic_arn: Settings.sns.topic_arn,
-      message: message
+      message: data_dir
```

Comment:
I wonder if we should have this be json, so we can send other fields in the future. I'm thinking that a request identifier would be good, so we can track things we kick off, but that never return.  Is that premature?

Reply:
Yeah, I thought about that but that would require that the lambda parse the message as JSON which is just an extra iota of complexity. Could always add later.